c10/core/ScalarType.h

```diff
@@ -269,7 +269,7 @@ static inline ScalarType promoteTypes(ScalarType a, ScalarType b) {
   // corrent values for the type promotions in complex type cases.
   static constexpr ScalarType _promoteTypesLookup[static_cast<int>(
       ScalarType::NumOptions)][static_cast<int>(ScalarType::NumOptions)] = {
-      /* u1  i1  i2  i4  i8  f2  f4  f8  c2  c4  c8  b1 */
+      /*        u1  i1  i2  i4  i8  f2  f4  f8  c2  c4  c8  b1 */
```

Comment:
Is this from autoformatting?

Reply:
Accidental commit -- I was double checking the promotions -- forgot to cleanup.